lua/lsp/servers/tsserver.lua

```diff
@@ -56,44 +56,42 @@ local on_attach = function(client, bufnr)
   require("lsp-inlayhints").on_attach(client, bufnr)
 end
 
+local function filter(arr, fn)
+  if type(arr) ~= "table" then
+    return arr
+  end
+
+  local filtered = {}
+  for k, v in pairs(arr) do
+    if fn(v, k, arr) then
+      table.insert(filtered, v)
+    end
+  end
+
+  return filtered
+end
+
+local function filterReactDTS(value)
+  -- Depending on typescript version either uri or targrtUri is returned
+  if value.uri then
+    return string.match(value.uri, '%.d.ts') == nil
+  elseif value.targetUri
+    return string.match(value.targrtUri, '%.d.ts') == nil
```

Comment:
`end` is missing

![image](https://user-images.githubusercontent.com/17691791/191561000-988369c3-aca6-4588-a82b-2bc294899ca9.png)

Reply:
+ `then` is missing in line 78

Reply:
Oh, good catch, sorry i missed those.